cypress/e2e/generate_prompt.cy.js

```diff
@@ -4,9 +4,86 @@ describe('Prompt generation', () => {
     const category_input = 'category input from cypress';
     const prompt_input = 'prompt input from cypress';
 
+    let userCreated = false;
+
     // The tests start by visiting the localhost URL for the application.
     beforeEach(() => {
         cy.visit('http://localhost:3000/');
+        if (!userCreated) {
+            userCreated = true;
+            cy.get('button:contains("Sign up")').first().click();
+            cy.get('span:contains("Username")')
+                .parent()
+                .children()
+                .filter('input')
+                .first()
+                .type('testuser', { force: true });
+            cy.get('span:contains("Password")')
+                .parent()
+                .children()
+                .filter('input')
+                .first()
+                .type('salasana', { force: true });
+            cy.get('span:contains("Repeat password")')
+                .parent()
+                .children()
+                .filter('input')
+                .first()
+                .type('salasana', { force: true });
+            cy.get('span:contains("Key")')
+                .parent()
+                .children()
+                .filter('input')
+                .first()
+                .type('DEV-123', { force: true });
```

Comment:
'DEV-123' should probably be retrieved from the dev key in .env to prevent future breaking if someone changes it.